web-app/src/index.js

```diff
@@ -35,10 +36,13 @@ const configureStore = () => {
 
 const isAuthenticated = () => {
   if (!authenticated()) {
-    return <Redirect to="/app" />;
+    if (window.location.pathname !== '/landingpage' && window.location.pathname !== '/develop' && window.location.pathname !== '/app' && window.location.pathname !== '/login' && window.location.pathname.substring(0, 25) !== '/login/callback/facebook/' && window.location.pathname !== '/reset-password' && window.location.pathname.substring(0, 16) !== '/reset-password/' && window.location.pathname !== '/sign-up' && window.location.pathname !== '/you-made-it' && window.location.pathname !== '/start') {
```

Comment:
This can be cleaned up by replacing:

```js
if (window.location.pathname !== '/landingpage' && window.location.pathname !== '/develop' && window.location.pathname !== '/app' && window.location.pathname !== '/login' && window.location.pathname.substring(0, 25) !== '/login/callback/facebook/' && window.location.pathname !== '/reset-password' && window.location.pathname.substring(0, 16) !== '/reset-password/' && window.location.pathname !== '/sign-up' && window.location.pathname !== '/you-made-it' && window.location.pathname !== '/start') {
    //
}
```

With:

```js
const urls = ['/landingpage', '/develop', '/app', '/login', '/login/callback/facebook', '/reset-password', '/sign-up', '/you-made-it', '/start'];
if (urls.some(url => window.location.pathname.includes(url))) {
    // 
}
```